src/Router.tsx

```diff
@@ -40,6 +42,68 @@ export const RouterInner = () => {
   const { sideMenuOpen, sideMenuMinimised, setContainerRefs } = useUi();
   const { t } = useTranslation('base');
 
+  const [urlNetworkExists, setUrlNetworkExists] = useState<boolean>(false);
+  const [urlNetwork, setUrlNetwork] = useState<any>('');
+  const [urlLngExists, setUrlLngExists] = useState<boolean>(false);
+  const [urlLng, setUrlLng] = useState<any>('');
+
+  // staking.polkadot.network?n=polkadot&l=fr'
+  useEffect(() => {
+    const urlNetworkSource = extractUrlValue('n');
+    if (
+      urlNetworkSource &&
+      (urlNetworkSource === 'polkadot' || 'kusama' || 'westend')
+    ) {
+      setUrlNetworkExists(true);
+      setUrlNetwork(urlNetworkSource);
+    }
+    const _network = urlNetworkExists
+      ? urlNetwork
+      : network
+      ? network.name.toLowerCase()
+      : DefaultNetwork;
```

Comment:
Something to be careful of is that `DefaultNetwork` is all lower case, whereas `NETWORKS[x].name is capitalised. We should resolve this and stick to one convention.